profiles/models.py

```diff
@@ -141,3 +141,16 @@ def can_edit_grid_element(self):
         if getattr(settings, "RESTRICT_GRID_EDITORS", False):
             return self.user.has_perm("grid.change_element")
         return True
+
+
+class ExtraFields(BaseModel):
+    profile = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    key = models.CharField(max_length=256)
+    value = models.URLField(max_length=256)
```

Comment:
What do you think about using these here? we are already using these in the forms as well?
```suggestion
    label = models.CharField(max_length=256)
    url = models.URLField(max_length=256)
```

Reply:
updates to field names made